### DISCUSSION TITLE
New Exploit (works for lightspeed, and maybe others?)

### DISCUSSION THREAD

**Discussion by axolotl8** [3kh0/ext-remover#1102] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I don't know if someone has already brought this up or not, but there is a method my friends and I at school discovered to bypass Lightspeed. Developer Mode (for extensions, not the other one), needs to be unblocked. Go to chrome://extensions, and ensure dev mode is on (top right corner). Click on the background page of the extension you want to block. Then, click on Network. While in Network, click Disable Cache. Also, click on No throttling, and be sure to change that to Offline. Once that is finished, head to Settings, and scroll down. Click on the Disable Javascript box. Please do not close this window (unless you want the extension to work again). Also, afterward, go to a website, do ctrl shift i, go to network, and switch it back to no disable cache, and no throttling.

**Comment by SIMONHSCH** [comment]
sorry ,I don't have dev-mode...

**Reply by Vhirttuegaming** [reply]
what is the background page?

**Comment by akabutnicer** [comment]
Not only can I assure you this has already been found by someone before, this simply will **not work for the majority of school -issued Chromebooks**.

**Reply by axolotl8** [reply]
ok
